docs/source/integrations/databricks.rst

```diff
@@ -0,0 +1,9 @@
+# Databricks integration
```

Comment:
This is now valid MyST (yay!) but will need a filename change to `.md`

Reply:
Done